data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -69,12 +69,15 @@
     "percent",
     "Percentage",
     "Energy burden",
-    "loss rate",
     "greater than or equal to 18 years",
     field_names.PERCENTILE_FIELD_SUFFIX,
 ]
-
 TILES_ROUND_NUM_DECIMALS = 2
+
+# FEMA rounding suffix
+FEMA_SUFFIX = "loss rate"
```

Comment:
I'm torn on this - on one hand it's consistent with how we handle `PERCENT_PREFIXES_SUFFIXES` above and so I appreciate the consistency. On the other it's a little nerve racking to have too much fancy logic based on string matching, for instance someone could add an "Energy loss rate" field and be surprised to see it rounded to 4 digits. So I'm tempted to suggest that we directly list and import the `[field_names.EXPECTED_POPULATION_LOSS_RATE_LOW_INCOME_FIELD, ...]` and check them exactly. 

Hopefully the Big Refactor we're talking about implementing will clean this up, where each field has a schema that defines its own float decimals. 

I could be convinced either way here (to stick with string matching or do exact matches on the field names). What do you think?

Reply:
Hmm... I don't have strong preferences here. I think clearer is better generally,  so I'll update this to list out the columns exactly, and I'll keep the `elif`